client/state/comments/test/actions.js

```diff
@@ -26,18 +25,14 @@ import {
 	unlikeComment
 } from '../actions';
 import {
-	getCommentParentKey,
 	createRequestId
 } from '../utils';
 import {
 	NUMBER_OF_COMMENTS_PER_FETCH
-} from '../constants'
-
-const MANY_COMMENTS_POST = {
-	siteId: 91750058,
-	postId: 287
-};
+} from '../constants';
 
+const SITE_ID = 91750058;
```

Comment:
Didn't asked myself that questions. Once we move this out to the data-layer and start dispatching plain objects for actions and wpcom-http actions for requests, it won't matter any more.